src/app/Dashboard/RecentResourceItem.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+// import { Link } from 'react-router-dom';
+
+
+export default function RecentResourceItem(props) {
```

Comment:
For function components, it's best practice to destructure the props object so it's clear in one place what props you'll be using.

```js
- export default function RecentResourceItem(props) {
+ export default function RecentResourceItem({ prop1, prop2, prop3 }) {
```

Reply:
this function now destructures props